eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js

```diff
@@ -0,0 +1,88 @@
+const {
+  ERR_RIGHTSIDE_AND_OR_NOT_ALLOWED,
+  ERR_RIGHTSIDE_ALLOFF_OR_NOT_ALLOWED,
+} = require("../../../constants/validationErrorCodes");
+
+const {
+  getOptionById,
+  getExpressionGroupByExpressionId,
+} = require("../../../schema/resolvers/utils");
+const { filter } = require("lodash");
+
+module.exports = function(ajv) {
+  ajv.addKeyword("validateExclusiveCheckbox", {
+    validate: function isValid(
+      otherFields,
+      entityData,
+      fieldValue,
+      dataPath,
+      parentData,
+      fieldName,
```

Comment:
And this one

Reply:
Well that's not helpful. I think `otherFields` and `fieldName` are the ones that aren't used